Mathlib/Data/Bool/Basic.lean

```diff
@@ -372,13 +375,15 @@ def toNat (b : Bool) : Nat :=
   cond b 1 0
 #align bool.to_nat Bool.toNat
 
+lemma toNat_le_one (b: Bool) : b.toNat ≤ 1 := by cases' b <;> simp
+
 /-- convert a `ℕ` to a `Bool`, `0 -> false`, everything else -> `true` -/
 def ofNat (n : Nat) : Bool :=
   decide (n ≠ 0)
 #align bool.of_nat Bool.ofNat
 
 theorem ofNat_le_ofNat {n m : Nat} (h : n ≤ m) : ofNat n ≤ ofNat m := by
-  simp only [ofNat, ne_eq, _root_.decide_not]
+  simp only [ofNat, ne_eq, _root_.decide_not];
```

Comment:
Can you undo the new semicolons in this file?

Reply:
Just one semicolon?